diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1,10 +1,14 @@
 //! PostgreSQL specific expression methods
 
+#[cfg(all(feature = "postgres_backend", feature = "serde_json"))]
```

Comment:
None of these traits should be coupled to the `serde_json` feature. You should be able to use them without enabling that feature, as you could use them purely in the generated SQL.

That applies to essentially all changes in this file.